src/qml/MapCanvas.qml

```diff
@@ -259,9 +262,11 @@ Item {
         mapArea.rightClicked(Qt.point(eventPoint.position.x, eventPoint.position.y), "touch");
       } else {
         if (!doublePressed) {
+          mapArea.clicked(point.position, "touch");
```

Comment:
We need to move that out into the timer, otherwise we'll end up triggering this when not wanting it.